#goal
float32 linear_pos
float32 launch_power
---
#result
bool success
---
#feedback